libraries/app/api.cpp

```diff
@@ -302,26 +302,30 @@ namespace graphene { namespace app {
        const auto& db = *_app.chain_database();
        FC_ASSERT( limit <= 100 );
        vector<operation_history_object> result;
-       const auto& stats = account(db).statistics(db);
-       if( stats.most_recent_op == account_transaction_history_id_type() ) return result;
-       const account_transaction_history_object* node = &stats.most_recent_op(db);
+       account_statistics_object stats;
+       try { stats = account(db).statistics(db); } catch(...) { return result; }
+       const account_transaction_history_object* node;
+       try { node = &stats.most_recent_op(db); } catch(...) { return result; }
```

Comment:
As mentioned earlier, we can merge this `try` with the previous one to avoid unnecessarily construction of `account_statistics_object` object (although it can be avoided by other means).